src/Esfa.Vacancy.Api.Types/Vacancy.cs

```diff
@@ -13,7 +13,7 @@ public class Vacancy
         /// <value>
         /// The reference.
         /// </value>
-        public int Reference { get; set; }
+        public int VacancyReference { get; set; }
```

Comment:
Now that we are going to use VacancyReferenceNumber every where, the API users shouldn't have any problem interpreting what Id stands for.